pom.xml

```diff
@@ -625,6 +626,15 @@
             <!-- License: Apache 2.0 -->
          </dependency>
 
+         <!-- needed by SyncCalculation -->
+         <!-- https://mvnrepository.com/artifact/org.hdrhistogram/HdrHistogram -->
+         <dependency>
+            <groupId>org.hdrhistogram</groupId>
+            <artifactId>HdrHistogram</artifactId>
+            <version>${org.hdrhistogram.version}</version>
+            <!-- License: Apache 2.0 -->
```

Comment:
I don't believe it is Apache license.

Reply:
Yep: super good point!! It is not obvious at all...I'm checking it!!!

Reply:
This one: https://github.com/HdrHistogram/HdrHistogram/blob/master/LICENSE.txt
Argh!!!

Reply:
Seems BSD 2 mmmmm

Reply:
Seems compatible...@clebert wdyt?